sort.h

```diff
@@ -19,4 +19,6 @@ typedef struct listint_s
 void print_list(const listint_t *list);
 void print_array(const int *array, size_t size);
 void bubble_sort(int *array, size_t size);
+void insert_node(listint_t **head_ptr, listint_t *node);
```

Comment:
I don't think there is a function called insert_node that you used. Possibly from the previous singly linkedlist that you implemented.